Modules/Loadable/Segmentations/Widgets/qMRMLSegmentsTableView.cxx

```diff
@@ -740,13 +748,20 @@ void qMRMLSegmentsTableView::setSelectedSegmentIDs(QStringList segmentIDs)
       {
       continue;
       }
+    validSelection = true;
     QItemSelectionModel::QItemSelectionModel::SelectionFlags flags = QFlags<QItemSelectionModel::SelectionFlag>();
     flags.setFlag(itemSelectionFlag);
     flags.setFlag(QItemSelectionModel::Rows);
     d->SegmentsTable->selectionModel()->select(index, flags);
     // Afther the first segment, we append to the current selection
     itemSelectionFlag = QItemSelectionModel::Select;
     }
+
+  // The list of segment IDs was either empty, or all IDs were invalid.
```

Comment:
Similar as above. If I just read this comment then it suggest that at this point, the list of segment IDs is empty. To make it very clear, either rephrase ("Check if...") or if we keep it as a statement then move it to the same line as the `if` or inside the branch.